CommunicationServiceTest/ServiceTests/ChatServiceTest.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using CommunicationService.Domain;
+using CommunicationService.Models;
+using CommunicationService.Repository;
+using CommunicationService.Services;
+using Moq;
+using Xunit;
+
+namespace CommunicationServiceTest.ServiceTests
+{
+    public class ChatServiceTest
+    {
+        private readonly IChatService _chatService;
+        private readonly Mock<IChatRepository> _repository;
+
+        public ChatServiceTest()
+        {
+            _repository = new Mock<IChatRepository>();
+
+            _chatService = new ChatService(
+                _repository.Object);
+        }
+
+        [Fact]
+        public async Task InitializeChat_Success_ReturnsCreatedChatObject()
+        {
+            var buyerId = Guid.NewGuid();
+            var sellerId = Guid.NewGuid();
+            var initModel = new InitializeModel()
+            {
+                Buyer = new User {Name = "test@test.test", Id = buyerId},
+                Seller = new User {Name = "my@email.be", Id = sellerId}
+            };
+            
+            var chatModel = new Chat()
```

Comment:
Waarom noemen we het een model als het geen model is. Alleen initModel is een model.
Ik zou het gewoon chat en returnChat noemen.